tss-esapi/src/abstraction/transient/mod.rs

```diff
@@ -106,11 +118,8 @@ pub struct ObjectWrapper {
 /// The `TransientKeyContext` makes use of a root key from which the other, client-controlled
 /// keyes are derived.
 ///
-/// Currently, only functionality necessary for RSA key creation and usage (for signing,
-/// verifying signatures, encryption and decryption) is implemented. The RSA SSA
-/// asymmetric scheme with SHA256 is used for all created and imported signing keys.
-/// The RSA OAEP asymmetric scheme with SHA256 is used for all created and imported
-/// signing/encryption/decryption keys.
+/// The main goal of this abstraction is to make public key cryptography more accessible,
```

Comment:
There is nothing wrong with the information in this section but it is something about how it is formulated... It is like it is expressing a vision rather then saying what it is actually does.